gcp-observability/src/main/java/io/grpc/gcp/observability/ObservabilityConfigImpl.java

```diff
@@ -100,19 +102,23 @@ private void parseConfig(Map<String, ?> config) {
         }
         this.eventTypes = eventTypesBuilder.build();
       }
-      String sampler = JsonUtil.getString(config, "global_trace_sampler");
       Double samplingRate = JsonUtil.getNumberAsDouble(config, "global_trace_sampling_rate");
-      checkArgument(
-          sampler == null || samplingRate == null,
-          "only one of 'global_trace_sampler' or 'global_trace_sampling_rate' can be specified");
-      if (sampler != null) {
-        this.sampler = new Sampler(SamplerType.valueOf(sampler.toUpperCase()));
+      if (enableCloudTracing && samplingRate == null) {
+        this.sampler = Samplers.probabilitySampler(0.0);
       }
+      double epsilon = 1e-6;
```

Comment:
Why not make this a class final constant with some comment (e.g. why `1e-6`)?

Reply:
Done